portal-web/test/functional/com/liferay/portalweb/paths/portlet/assetlists/AssetLists.path

```diff
@@ -80,8 +96,8 @@
 	<td></td>
 </tr>
 <tr>
-	<td>PERSONALIZATION_DROPDOWN</td>
-	<td>//div[contains(.,'Anyone')]//*[contains(@class,'dropdown-toggle')]</td>
+	<td>PERSONALIZATION_TITLE</td>
```

Comment:
This path was on macro.gotoPersonalizationMenuItem(), but this macro was never used and when I tried to use it, was broke.